frontend/angular.json

```diff
@@ -149,6 +149,17 @@
                             "buildTarget": "digital-fuesim-manv:build:development"
                         }
                     },
+                    "options": {
+                        "prebundle": {
+                            "exclude": [
+                                "digital-fuesim-manv-shared",
+                                "@noble/hashes/sha256",
+                                "@noble/hashes/crypto",
+                                "rbush-knn",
+                                "tinyqueue"
```

Comment:
Uh, I don't like having to keep such a list here. But I guess there is no other way to achieve this. And at least it will yell quite extensive at the developer when a *new* dependency has to be added here.